backend/settings.py

```diff
@@ -15,6 +15,8 @@
 import sys
 import backend.constants as constants
 
+_isProd = os.environ.get('PROD') == 'true'
```

Comment:
why the underscore prefix? Naming conventions suggest that this should be `IS_PROD`. Instead of a whole new environment variable can production be derived from other environment variables? like, `IS_PROD = not DEBUG`? or `IS_PROD = PARITY_ENV == 'parity'`? IDK if deriving it like that is actually any better.

Reply:
I felt like having DEBUG derive from IS_PROD would make more sense. I was imagining _isProd as a private variable but I realize now that's kinda confusing. I'll change the naming convention.